mongo/src/main/java/com/redhat/lightblue/mongo/crud/IterateAndUpdate.java

```diff
@@ -48,28 +59,39 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IterateAndUpdate.class);
 
+    private final int batchSize = 64;
```

Comment:
MongoMetadataConstants maybe? At least then we wouldn't loose them later when we do go to make it configurable.